src/main/java/org/jabref/model/entry/FileFieldWriter.java

```diff
@@ -0,0 +1,66 @@
+package org.jabref.model.entry;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileFieldWriter {
+
+    private FileFieldWriter() {
+    }
+
+    public static String getStringRepresentation(List<LinkedFile> fields) {
+        String[][] array = new String[fields.size()][];
+        int i = 0;
+        for (LinkedFile entry : fields) {
+            array[i] = new String[] {entry.getDescription(), entry.getLink(), entry.getFileType()};
+            i++;
+        }
+        return encodeStringArray(array);
+    }
+
+    public static String getStringRepresentation(LinkedFile field) {
+        return getStringRepresentation(Collections.singletonList(field));
+    }
+
+    /**
+     * Encodes a two-dimensional String array into a single string, using ':' and
+     * ';' as separators. The characters ':' and ';' are escaped with '\'.
+     * @param values The String array.
+     * @return The encoded String.
+     */
+    public static String encodeStringArray(String[][] values) {
```

Comment:
It is public because it is used somewhere else. You are probably right and this method should be private, but I have no idea what "FileListTableModel" is doing...